tests/python/gaia-ui-tests/gaiatest/apps/messages/app.py

```diff
@@ -37,10 +37,12 @@ def tap_create_new_message(self):
         from gaiatest.apps.messages.regions.new_message import NewMessage
         return NewMessage(self.marionette)
 
-    def tap_options(self):
```

Comment:
Options is still used in the application if you have at least 1 message in the application. But as we're dropping the support and no other tests seem to use this function, I'm fine with removing it.